content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -328,3 +348,6 @@ On the APM Service Page, view the direct downstream database dependencies of the
 [16]: https://www.nuget.org/packages/npgsql
 [17]: https://www.nuget.org/packages/MySql.Data
 [18]: https://www.nuget.org/packages/MySqlConnector
+[19]: https://github.com/DataDog/dd-trace-dotnet
```

Comment:
more C&P (at Pierres request: ;-))

Reply:
What? We don't do .NET? :p

Reply:
We absolutely do: https://php.net/Dotnet